index.js

```diff
@@ -70,3 +72,33 @@ app.post('/submit', async (req, res) => {
 
     res.json({message: 'Data succesfully saved!', note: result.rows[0]})
 })
+
+app.post('/createUser', async(req, res) => {
+  const {username, password} = req.body;
+  try{
+    const queryCheck = 'SELECT * FROM account WHERE user_name = $1';
+    const valueCheck = [username];
+    const resultCheck = await client.query(queryCheck, valueCheck);
+    if(resultCheck.rows.length > 0 ){
+      return res.status(400).json({error: 'Username already exists.'});
+    }else{
+      const hashedPassword = await bcrypt.hash(password, saltRounds);
```

Comment:
saltrounds is a hardcoded 5, which is pretty low. thats ok but i think you can also leave it empty, and that will default to 10. this is because we dont like hard coded numbers

Reply:
check !fixup